### DISCUSSION TITLE
How to prevent use risky npm packages which scanned by osv-scanner in bun?

### DISCUSSION THREAD

**Discussion by dlintw** [oven-sh/bun#9422] [General]
For example, [osv-scanner](https://github.com/google/osv-scanner) scan [gitea](https://github.com/go-gitea/gitea)'s package-lock.json.

1. bun's lock file can not be scan by osv-scanner(maybe require someone to patch that).
2. Any option of bun can be enable to prevent using those risky package?
3. How to solve those risky package if they exists in deep dependency. Can I manually solve those in bun's env when the upstream already dead or I require fix them ASAP?

**Comment by ghost** [comment]
Hey @dlintw - although not directly your usecase you still might find this helpful:

https://bun-osv-scanner.com/
